### PR TITLE
Initial Rebrand Effort

### PR DESCRIPTION
**Pull request by Spiker985**
Large PR - Individual commits per associated segment

Rebrand all of the non-api/non-variable/non-github link instances of Atom or atom to Pulsar or pulsar respectively

Current build commands
```bash
nvm install 16.16.0 #May not be needed
npm install --location=global yarn
yarn install
yarn build
yarn build:apm #Not strictly necessary
yarn start
```

Known bugs as of initial PR:
- GitHub renderer.html/renderer.js require is not defined
- Tree-sitter is an incompatible version. It wants 13, gets 10
- Settings-View package-manager fails to fetch local or outdated packages/themes - This is presumed to be a problematic interaction with apm. Apm expects files to be in a `.atom` directory, which is no longer the case.

### CODE REVIEW

**Comment by Spiker985** [comment]
Follow up commits planned:
- Migrate instances of Pulsar to use the branding api where possible

**Comment by confused-Techie** [comment]
This is a huge commit with lots of fantastic work. I'll try to dig deeper to see if theres any issues but otherwise looks great. But feel we should go ahead and try to use the branding API where possible before merging, but otherwise fantastic work

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: CI
```
Comment (mauricioszabo) [inline .github/workflows/main.yml:1]: I think we kinda duplicated this file on this PR: https://github.com/pulsar-edit/pulsar/blob/master/.github/workflows/editor-tests.yml

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: CI
```
Comment (mauricioszabo) [inline .github/workflows/ci.yml:1]: We already have tests running on GH actions now, maybe we can remove this file, WDYT?

**Comment by mauricioszabo** [comment]
I'm going to close this PR so we can focus on https://github.com/pulsar-edit/pulsar/pull/22 :)

**Pull request closed without merging**